arclightning_backend/src/router.rs

```diff
@@ -143,4 +143,27 @@ impl Router {
             _ => self.invalid_endpoint(),
         }
     }
+
+   
 }
+
+fn list_files(dir: PathBuf) -> Result<Vec<PathBuf>, io::Error>{
+    
+    if dir.is_dir(){
+        
+        let  paths: Vec<PathBuf> = fs::read_dir(dir)?.flatten()
+        .map(|entry| entry.path()).flat_map(list_files(&entry)).flatten().collect();
+        Ok(paths)
+
+        
+    }else{
+        let paths: Vec<PathBuf> ;
```

Comment:
A more concise shortcut of initializing a new vector here would be `vec![dir]`.  Then you don't have to spend 2 lines doing so